in-depth/submission_guidelines.md

```diff
@@ -0,0 +1,177 @@
+---
+layout: docs
+title: Theme & Plugin Submission Guidelines
+headline: Theme & Plugin Submission Guidelines
+description: |
+  Submitting your Theme or Plugin for inclusion on the website is very easy.<br>
+  We've written some guidelines here to walk you through the process.
+toc:
+  as-easy-as-making-a-pull-request: As Easy as Making a Pull Request
+  the-file-layout: The File Layout
+  screenshot-specifics: Screenshot Specifics
+  metadata-breakdown: Metadata Breakdown
+  markdownyaml-template: Markdown/YAML Template
+nav-url: /docs/
+---
+
+## As Easy as Making a Pull Request
+
+Submitting content to the website is easy, all you have to do is create a new [Pull Request][] on GitHub.  To make the submission process smoother, our [Themes][] and [Plugins][] pages are dynamically generated using a collection of Markdown files.  The guidelines on this page will explain the layout and structure those files.  While we don't anticipate having to turn anyone down, we do reserve the right to reject your submission at our own discretion.  More likely though, we will work with you to get your submission up to our satisfaction.
```

Comment:
I found the *second-to-last* sentence too harsh, that's why I tried to soften it a little with the last one. :unamused:

I'm still not happy with it either though, so I'll work on it.